qlty-plugins/plugins/tests/runLinterTest.ts

```diff
@@ -12,7 +12,8 @@ Debug.inspectOpts!.hideDate = true;
 
 // Currently unsupported tools on Windows
 const SKIP_LINTERS = {
-  win32: ["semgrep", "swiftlint", "dockerfmt"],
+  win32: ["semgrep", "swiftlint", "dockerfmt", "stringslint"],
+  linux: ["stringslint", "swiftlint"],
```

Comment:
Not for this PR, but I wonder if we can infer SKIP_LINTERS behavior from the `supported_platforms` data someday